pkgs/os-specific/linux/bolt/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, meson, ninja, pkgconfig, fetchFromGitLab,
+  python3, umockdev, gobject-introspection,
+  git, asciidoc, libxml2, libxslt, docbook_xml_dtd_45, docbook_xsl,
+  glib, systemd, polkit
+}:
+
+stdenv.mkDerivation rec {
+  name = "bolt-${version}";
+  version = "0.7";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.freedesktop.org";
+    owner = "bolt";
+    repo = "bolt";
+    rev = "${version}";
+    sha256 = "0xn2c31kcjh1j76gq1qrcxwjyjyqnsxygkfrvh3xk07qc92f99xd";
+  };
+
+  nativeBuildInputs = [
+    meson ninja pkgconfig umockdev
+    git asciidoc libxml2 libxslt docbook_xml_dtd_45 docbook_xsl
+    (python3.withPackages
+      (p: [ p.pygobject3 p.dbus-python p.python-dbusmock ]))
+    gobject-introspection
+  ];
+
+  buildInputs = [
+    glib systemd polkit
+  ];
+
+  postPatch = ''
+    echo '#!/bin/sh' > scripts/meson-install.sh
```

Comment:
I would prefer removing [this line](https://gitlab.freedesktop.org/bolt/bolt/blob/00b3d8a35835b8c856e5de026a0c8316b2ceba90/scripts/meson-install.sh#L11) in a patch so that we do not accidentally ignore something when new commands are added to that file. Or at least add a comment that the script is trying to create `/var/lib/boltd`.

Reply:
Will move to a patch.